python/ray/monitor.py

```diff
@@ -269,8 +266,8 @@ def _run(self):
         # Initialize the mapping from raylet client ID to IP address.
         self.update_raylet_map()
 
+        self.get_all_heartbeat()
         # Initialize the subscription channel.
-        self.psubscribe(ray.gcs_utils.XRAY_HEARTBEAT_BATCH_PATTERN)
         self.psubscribe(ray.gcs_utils.XRAY_JOB_PATTERN)
```

Comment:
Wondering if we should just remove the pubsub client entirely now that it isn't really used. It seems the job handler just prints a message.